src/rich-text-editor.component.js

```diff
@@ -45,7 +46,6 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
   useEffect(() => {
     if (props.save && props.unchangedInterval && divRef.current && focused) {
-      setTimeout(save, props.unchangedInterval)
```

Comment:
I think this was old code that was never removed. Don't need it to be continually firing like this

Reply:
yeah i think you're right -- looks like this wasn't needed. Also, I think the unchangedTimeout could also probably not be a ref and just be a variable in the scope of the effect. So if you're feeling ambitious, you could make that change, but I don't think it's necessary